reesegreen2014/TaskPro
# Feature/addtaskform-priority-levels

What does this PR do?

This PR updates the TaskCard component to include priority levels with distinct colors and descriptions. It also enhances the styling of task cards for better visual consistency and responsiveness.

Where should the reviewer start?

Review the TaskCard.jsx component to understand the changes made to include priority levels. Check the TaskCard.css file for updates on the styling and responsiveness improvements.

How should this be manually tested?

Ensure that the priority levels are correctly displayed with the appropriate colors and descriptions on each task card. Verify that the task cards are responsive and that the "View Details" button looks like a button and navigates to the task details page.

Any background context you want to provide?

The addition of priority levels helps users quickly identify tasks based on their urgency. Styling adjustments aim to improve the overall user experience by making the task cards more consistent and visually appealing.

What are the relevant tickets (if any)?

No specific tickets are referenced for this change.

Screenshot:
<img width="1295" alt="Screenshot 2024-07-21 at 12 01 01 AM" src="https://github.com/user-attachments/assets/fddc604f-8d6e-4d11-86a0-76c13f2e10bd">
